code/modules/clothing/rogueclothes/pants.dm

```diff
@@ -111,19 +111,6 @@
 	salvage_result = /obj/item/natural/hide/cured
 	can_hold_endowed = TRUE
 
-/obj/item/clothing/under/roguetown/trou/leather
-	name = "leather trousers"
-	desc = "Trousers made of fine leather."
-	icon_state = "leathertrou"
-
```

Comment:
Why are leather trousers and skirts just removed?

Reply:
Moved to leather file

Reply:
Aah I see thank you